cnxauthoring/storage/postgresql.py

```diff
@@ -61,6 +61,7 @@ class PostgresqlStorage(BaseStorage):
     def __init__(self, db_connection_string=None):
         #initialize db
         self.conn = psycopg2.connect(db_connection_string)
+        self.db_connection_string = db_connection_string
```

Comment:
adding a variable to store the db_connection string needed to restart the database when a connection is broken or lost.